modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/impl/ItIdempotentCommandCacheTest.java

```diff
@@ -179,11 +195,22 @@ private static class Node implements AutoCloseable {
 
             cmgManager = mock(ClusterManagementGroupManager.class);
 
-            Path metaStorageDir = workDir.resolve("metastorage" + index);
+            ComponentWorkingDir metastorageWorkDir = new ComponentWorkingDir(workDir.resolve("metastorage" + index));
+
+            msLogStorageFactory =
+                    SharedLogStorageFactoryUtils.create(clusterService.nodeName(), metastorageWorkDir.raftLogPath());
+
+            RaftOptionsConfigurator msRaftConfigurator = options -> {
+                RaftGroupOptions raftOptions = (RaftGroupOptions) options;
+
+                // TODO: use interface, see https://issues.apache.org/jira/browse/IGNITE-18273
+                raftOptions.setLogStorageFactory(msLogStorageFactory);
+                raftOptions.serverDataPath(metastorageWorkDir.metaPath());
+            };
```

Comment:
This spell seems to be repeated a lot. How about extracting a method with 2 parameters and using it instead?